src/logic/community/communities.ts

```diff
@@ -281,6 +328,21 @@ export function createCommunityComponent(
         }
       }
 
+      try {
```

Comment:
We are already catching inside of `sendManualReviewNotification` function we can remove this one